graphics/vulkan-tools/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        KhronosGroup Vulkan-Tools v1.1.112
+name                vulkan-tools
+version             v1.1.112
```

Comment:
Same comment as above. Remove the `version` line entirely and use this instead:

`github.setup        KhronosGroup Vulkan-Tools 1.1.112 v
`